dashboard/test/ui/support/dashboard_helpers.rb

```diff
@@ -1,7 +1,11 @@
 module DashboardHelpers
   # Requires the full rails environment. Use sparingly, known to take 20-30s.
   def require_rails_env
+    puts 'Requiring rails env'
+    start = Time.now
     require File.expand_path('../../../../config/environment.rb', __FILE__)
+    finish = Time.now
+    puts "Requiring rails env took #{finish - start} seconds"
```

Comment:
Is this intending to being merged? If so, I'm opposed to `puts` statements. Use the LOGGER or message Slack or otherwise log the information.